lib/uplink/project.go

```diff
@@ -151,6 +152,11 @@ func (p *Project) GetBucketInfo(ctx context.Context, bucket string) (b storj.Buc
 func (p *Project) OpenBucket(ctx context.Context, bucketName string, access *EncryptionAccess) (b *Bucket, err error) {
 	defer mon.Task()(&ctx)(&err)
 
+	err = p.CheckBucketAttribution(ctx, bucketName)
+	if err != nil {
+		return nil, err
```

Comment:
more a question than comment @egonelbre when should we wrap errors and when should we just return them?

Reply:
When in doubt Wrap.

There are minor cases where we don't want to hide the underlying error, which is where we skipped the wrapping. With `errs.IsFunc` and `errs.Is` we should be able to fix those situations.